BetterMD/elements/symbol.py

```
@@ -20,20 +21,28 @@ class Symbol:
+        super().__init_subclass__(**kwargs)
+
+    def __init__(self, styles:'dict[str,str]'={}, classes:'list[str]'=[], inner:'list[Symbol]'=[], **props):
         self.styles = styles
         self.classes = classes
         self.children = list(inner) or []
         self.props = props
-        self.dom = dom
-        
+
+
     def copy(self, styles:'dict[str,str]'={}, classes:'list[str]'=[], inner:'list[Symbol]'=None):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)